test/test-hook.js

```diff
@@ -4,7 +4,7 @@ import fs from 'fs'
 import path from 'path'
 import { equal } from 'assert';
 
-import hook from '../';
+import hook from '../src';
```

Comment:
Need to include actual sources to get proper ES6 coverage, not the coverage of compiled code.